redisinsight/api/src/modules/redis/client/node-redis/cluster.node-redis.client.ts

```diff
@@ -67,6 +67,19 @@ export class ClusterNodeRedisClient extends NodeRedisClient {
     );
   }
 
+  /**
+   * @inheritDoc
+   */
+  async sendMulti(commands: RedisClientCommand[]): Promise<Array<[Error | null, RedisClientCommandReply]>> {
+    return Promise.all(
+      commands.map(
+        (cmd) => this.sendCommand(cmd)
+          .then((res): [null, RedisClientCommandReply] => [null, res])
+          .catch((e): [Error, null] => [e, null]),
+      ),
+    );
+  }
```

Comment:
I'm not sure that this realization is correct.
Could you test this with 2 commands? 
1. `set foo bar`
2. `lpush foo 1`
on both ioredis and node-redis clients and compare:
1. responses
2. data (key "foo") inside redis database